utils/location.ts

```diff
@@ -1,8 +1,17 @@
 export const isLocation = () => {
```

Comment:
`isLocation`, `isblock` 두 개의 함수가 하는 일이 비슷해 보이네요. 굳이 유틸함수로 만들기 보다는 아래처럼 사용하는 곳에서 변수로 만들어 사용하셔도 무난할 것 같습니다 🤗
```suggestion
const location = window.location.pathname;
const isLocation = location.includes("folder")
```